pytest/tests/shardnet/scripts/create_account.sh

```diff
@@ -5,5 +5,5 @@ account_id=$(grep account_id /home/ubuntu/.near/shardnet/validator_key.json | aw
 mkdir -p /home/ubuntu/.near-credentials/shardnet/
 echo "{\"account_id\":\"near\",\"public_key\":\"$1\",\"private_key\":\"$2\"}" > /home/ubuntu/.near-credentials/shardnet/near.json
 pk=$(grep public_key /home/ubuntu/.near/shardnet/validator_key.json | awk -F'"' '{print $4}')
-cp /home/ubuntu/.near/shardnet/validator_key.json /home/ubuntu/.near-credentials/shardnet/$account_id.json
+cp /home/ubuntu/.near/shardnet/validator_key.json /home/ubuntu/.near-credentials/shardnet/${account_id}.json
```

Comment:
Braces don’t matter here.  You should quote the variable though:
```suggestion
cp /home/ubuntu/.near/shardnet/validator_key.json /home/ubuntu/.near-credentials/shardnet/"$account_id".json
```